docs/readthedocs/source/doc/PPML/Overview/attestation_basic.md

```diff
@@ -71,10 +71,20 @@ With this attestation service design, we can avoid adding malicious applications
 
 #### Attestation Service for Cloud Service Provider (CSP)
 
+Before we submit our applications to a cloud service, we need to verify the identity and security posture of the platform. Azure Attestation receives evidence from the platform, validates it with security standards, evaluates it against configurable policies, and produces an attestation token for claims-based applications.
+
+The involved actors in Azure Attestation workflow:
+* Relying party: The component which relies on Azure Attestation to verify enclave validity.
+* Client: The component which collects information from an enclave and sends requests to Azure Attestation.
+* Azure Attestation: The component which accepts enclave evidence from client, validates it and returns attestation token to the client
+
+![Azure Attestation Workflow](../images/azure_attestation_workflow.png)
```

Comment:
Can we directly refer azure's image instead of import it to our repo?

Reply:
Refined.